wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java

```diff
@@ -352,6 +352,12 @@ public void removeWorkspace(String workspaceId) throws ConflictException, Server
      * @param accountId
      *         account which should be used for this runtime workspace or null when
      *         it should be automatically detected
+     * @param isRestore
+     *         if <code>true</code> workspace will be restored from snapshot if snapshot exists,
```

Comment:
It is not clear from the declaration what happens if there is no snapshot available.